docs/datasets_and_tables/transitdatabase.md

```diff
@@ -33,59 +33,92 @@ The data in the Airtable Transit Database is distinct but related to the followi
+| `gtfs service data` <br><br> *Primary Field*: `Name`, a combination of a set of `Services` record(s) and a single `gtfs datasets` record. | Each record links together a single `gtfs dataset` and one (if possible) or more `services`.  Additional fields define how to isolate the service within the `gtfs dataset`.  <br><br>Many services have more than one GTFS dataset which describes their service. Often these are either precursors to *final* datasets (e.g. AC Transit's GTFS dataset is a precursor to the Bay Area 511 dataset) or artifacts produced in other processes such as creating GTFS Realtime (e.g. [the VCTC GTFS produced by GMV](https://airtable.com/appPnJWrQ7ui4UmIl/tblnVt5FZ2FZmDjDx/viw3NtcDP3Qm0BYyG/recJhrNj21mYVETJG?blocks=hide)). The property `Category` indicates if this is the `primary` dataset, a `precursor` or `unknown` in order to distinguish which should be used. See |
 | `place geography` <br><br> *Primary Field*: `Name` | Each place is a Census recognized Place with a FIPS code.  Additional properties include County and Caltrans District. |
+| `county geography` <br><br> *Primary Field*: `Name` | Each record is a county and has fields to lookup key information about that county such as Caltrans District. |
 | `fare systems`  <br><br> *Primary Field*: `Fare System` | `WIP` A list of fare systems and their properties. Fare systems apply to one or more Organization which manages a service. |
 | `funding programs` <br><br>*Primary Field*: `Program` | `WIP` A very broad list of funding programs for `services` such that we are able to identify which `services` may be subject to various requirements and/or to better classify `services`. |
 | `rider requirements` <br><br> *Primary Field*: `Requirement` | A very broad list of rider requirements to categorize and analyze `services`. |
 | `eligibility programs` <br><br> *Primary Field*: `Program` | `WIP` Each record is a program/process which riders must use to become eligible to ride one or more `services`.  Each program is operated by an `organization` and evaluates one or more `rider requirements`.  |
+| `service-component` | Imported table from [Transit Technology Stacks](#transit-technology-stacks) which allows us to look at what products services use. |
+| `feed metrics` | Metrics (i.e. service area, population, etc. ) that are calculated by Data Analysts using data warehouse that are useful for planning and strategy.  **Not necessarily up-to-date; should be used as back of the envelope numbers onl; not for presentation.** |
```

Comment:
Small typo -- "onl" instead of "only" in `feed metrics`

Reply:
fixed!